substrate/client/transaction-pool/src/graph/validated_pool.rs

```diff
@@ -666,11 +666,28 @@ impl<B: ChainApi> ValidatedPool<B> {
 		self.listener.write().retracted(block_hash)
 	}
 
+	//todo: doc + rename!
```

Comment:
Will this be covered as part of this PR?

Reply:
```suggestion
```